dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -50,4 +50,13 @@ def search
     end
     render json: search_results
   end
+
+  # GET /dashboardapi/v1/schoolzipsearch/:q/:limit
+  def zip_search
+    search_results = Api::V1::SchoolAutocomplete.get_zip_matches(
```

Comment:
Do we even need to use SchoolAutocomplete? Or can this just be `School.where(zip: params.require(:q)`? (Relatedly, can `q` change to `zip`?)

Reply:
Followup- do we want or need a limit anymore? If we're already limited by zip code I'm inclined to remove it